features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -1913,6 +1913,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                                     getAllowedList(metadata?.fapiMetadata?.tokenEndpointAuthMethod)
                                                     : getAllowedList(metadata?.tokenEndpointAuthMethod) }
                                                 readOnly={ readOnly }
+                                                data-componentId={ `${ testId }-client-auth-method-dropdown` }
```

Comment:
`testId` shall we replace this with `componentid` ?, check other places as well.